manual_tests.py

```diff
@@ -1,61 +1,89 @@
 # file for manual local tests
+import peppy
```

Comment:
What are `manual_tests.py`? Are those for yourself? If so maybe we can `.gitignore` them.

Reply:
I am running this script to check if everything works correctly, especially SQL queries. Because mock tests are not checking it

Reply:
I like including manual tests in there, it's nice for me. So I'd not .gitignore them.